cmd/deploy/deploy.go

```diff
@@ -513,6 +513,17 @@ func getDefaultTimeout() time.Duration {
 
 func GetDeployer(ctx context.Context, manifest *model.Manifest, opts *Options, builder *buildv2.OktetoBuilder, cmapHandler configMapHandler) (deployerInterface, error) {
 
+	isRunningRemotelyEnvVar := utils.LoadBoolean(constants.OktetoForceRemote)
+
+	if opts.Manifest != nil && opts.Manifest.Deploy != nil {
+		if !isRunningRemotelyEnvVar || opts.Manifest.Deploy.Remote == true {
+			opts.RunInRemote = true
+		}
+	} else {
+		err := oktetoErrors.ErrCouldNotInferAnyManifest
+		panic(err)
```

Comment:
Returning errors instead of `panic` is the best approach in general, I think in this case the error can easily be returned upstream

Reply:
Done!